Mathlib/Probability/Moments.lean

```diff
@@ -282,6 +287,27 @@ theorem iIndepFun.cgf_sum {X : ι → Ω → ℝ}
   · rw [h_indep.mgf_sum h_meas]
   · exact (mgf_pos (h_int j hj)).ne'
 
+theorem mgf_ident_distrib (X Y : Ω → ℝ) (hident : IdentDistrib X Y μ μ) :
+  mgf X μ t = mgf Y μ t := by
```

Comment:
We have quite rigid indentation rules in mathlib (it may look cumbersome at first, but in the long run it helps to have a coherent library). So, except for the first one, the lines of the statement should be indented by 4 spaces, while the lines of proofs should be indented by 2 spaces (plus additional indentation for subgoals). Here, you do the opposite :-)